Library/Homebrew/cxxstdlib.rb

```diff
@@ -1,23 +1,30 @@
-# typed: true # rubocop:todo Sorbet/StrictSigil
+# typed: strong
 # frozen_string_literal: true
 
 require "compilers"
 
 # Combination of C++ standard library and compiler.
 class CxxStdlib
+  sig { params(type: T.nilable(Symbol), compiler: T.any(Symbol, String)).returns(CxxStdlib) }
```

Comment:
If preferred, could make this `compiler: Symbol` at least based on our usage of it. We currently do `compiler.to_sym` in constructor so had let `String` be included, but our use case is in `tab#cxxstdlib` which already converts to symbol.

https://github.com/Homebrew/brew/blob/aed28da008de96a8cd82efcc763f06725698ffa6/Library/Homebrew/tab.rb#L361